components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -34,6 +34,17 @@ export default function TeamBilling() {
     const [pendingTeamPlan, setPendingTeamPlan] = useState<PendingPlan | undefined>();
     const [pollTeamSubscriptionTimeout, setPollTeamSubscriptionTimeout] = useState<NodeJS.Timeout | undefined>();
 
+    console.log(
+        "members",
+        members.length,
+        "currency",
+        currency,
+        "teamSubscription",
+        teamSubscription,
+        "pendingTeamPlan",
+        pendingTeamPlan,
+    );
+
```

Comment:
**suggestion**: Probably not the best line I could comment about this, but what do you think of making the [section title](https://github.com/gitpod-io/gitpod/pull/10182/files#diff-fab4237d49423136bbbf60168ab7b05684291690d3b0679ae4fa02d51c858f55R157) more actionable and relatable to the action below?

From _No billing plan_ to _Upgrade team plan_. 💡